boot_mode_helper.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
```

Comment:
Please move this file to the `JetPack/src/pilot` directory.

Reply:
Yeah, Actually I got this after pushing the commit. will update